easybuild/easyconfigs/p/pestpp/pestpp-5.0.5-foss-2020a.eb

```
@@ -0,0 +1,54 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	Custom
+#
+# Notes::
+##
+
+easyblock = 'CMakeMakeCp'
+
+name = 'pestpp'
+version = '5.0.5'
+github_account = "usgs"
+
+homepage = 'https://github.com/usgs/pestpp'
+description = """PEST++ is a software suite aimed at supporting
+ complex numerical models in the decision-support context.
+ Much focus has been devoted to supporting environmental models
+ (groundwater, surface water, etc) but these tools are readily
+ applicable to any computer model.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['b9695724758f69c1199371608b01419973bd1475b1788039a2fab6313f6ed67c']
+
+builddependencies = [
+    ('CMake', '3.16.4'),
+]
+
+# srcdir = 'src'
+
+build_type = 'Release'
+
+files_to_copy = [
+    (['bin/linux/*'], 'bin'),
+]
+
+sanity_check_paths = {
+    'files': [
+        'bin/pestpp-glm',
+        'bin/pestpp-sen',
+        'bin/pestpp-opt',
+        'bin/pestpp-ies',
+        'bin/pestpp-pso'],
```

this can be condensed to:
```suggestion
    'files': ['bin/pestpp-%s' % x for x in ['glm', 'sen', 'opt', 'ies', 'pso']],
```